hoomd/HOOMDMath.h

```diff
@@ -492,20 +492,20 @@ inline HOSTDEVICE void sincospi(double x, double& s, double& c)
     #endif
     }
 
-//! Compute the pow of x,y
+//! Compute the pow of x,y with single precison via exp2(log2) refactoring
```

Comment:
As Vyas said, this should document that this approximation is invalid for negative x. I don't think the loss of generality is a problem. Developers that use the fast math functions have to opt-in to them and should understand the limitations before doing so.

Reply:
I also added a note that this redefinition is undefined for negative bases for the sake of clarity.